core/common/math/math.hpp

```diff
@@ -10,6 +10,7 @@
 
 namespace fc::common::math {
   using primitives::BigInt;
+  using primitives::go::bitlen;
```

Comment:
move it to core/primitives/go/math.hpp, line 35

Reply:
fixed